web/scripts/display_yaml.js

```diff
@@ -0,0 +1,14 @@
+// Input parametes for creating the policy yaml
+
+let source_label = {app: "test"};
+let target_label = {app: "final"};
```

Comment:
there are chances that we are going to select set of labels for target and what id the source already has set of labels.

Reply:
would this work .?